webapp/src/actions.js

```diff
@@ -0,0 +1,31 @@
+import PluginId from './plugin_id';
+import {STATUS_CHANGE, OPEN_ROOT_MODAL, CLOSE_ROOT_MODAL} from './action_types';
+
+export const openRootModal = () => (dispatch) => {
+    dispatch({
+        type: OPEN_ROOT_MODAL,
+    });
+};
+
+export const closeRootModal = () => (dispatch) => {
+    dispatch({
+        type: CLOSE_ROOT_MODAL,
+    });
+};
+
+export const mainMenuAction = openRootModal;
+export const channelHeaderButtonAction = openRootModal;
+
+export const getStatus = () => (dispatch) => {
+    fetch('/plugins/' + PluginId + '/').then((r) => r.json()).then((r) => {
```

Comment:
I think I'm going to expose a selector in Redux to simplify operations around using the `PluginId`, e.g. building a path to the HTTP API, or resolving websocket events, etc.